zarafa/__init__.py

```diff
@@ -1439,7 +1439,7 @@ def container_class(self, value):
     def item(self, entryid):
         """ Return :class:`Item` with given entryid; raise exception of not found """ # XXX better exception?
 
-        item = Item() # XXX copy-pasting..
+        item = Item(parent=self) # XXX copy-pasting..
```

Comment:
okay, so this should work and make things faster. note that in 7.2, we renamed 'parent' into 'folder' for clarity.